sdk/lib/3rdparty/freetype/src/base/ftdebug.c

```diff
@@ -87,9 +91,22 @@
             int          line,
             const char*  file )
   {
+#if 0
+    /* activating the code in this block makes FreeType very chatty */
```

Comment:
Is it because of the setjmp/longjmp being used?

Reply:
This line is freetype original code.